### DISCUSSION TITLE
유니티 완전 초보) 롤러코스터 같은걸 만들고 싶습니다.

### DISCUSSION THREAD

**Discussion by dydgnlkim** [naverz/zepeto-studio-kor#935] [5. Q&A]
오브젝트를 자유자제로 움직이고 해당 지점에서도착하는 로직을 짜고 싶어서 이것저것 찾아봤는데

Slerp 를 사용해 나선형은 해봤는데 너무 모르겠네요...ㅠㅠ 

처음은 간단하게 원형 형태로 오브젝트가 한바퀴를 간단하게 돌고 멈추고를 구현하고싶습니다. 도움 부탁드려요.. ㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 무빙 블록 샘플이 도움이 될 것 같습니다.
https://github.com/naverz/zepeto-studio-kor/discussions/902

**Reply by dydgnlkim** [reply]
제가 말씀드린건 롯데월드에 아틀란티스처럼 해당 오브젝트가 레일에서 위치값 들을 자연스럽게 움직이게 하는 방법이 뭔지 궁금해서 여쭤봤습니다

**Reply by Deytooh** [reply]
@dydgnlkim Spline Curve / Bezier Curve 이란 거 쓰면 돼요 !

**Reply by dydgnlkim** [reply]
@Deytooh 기존에 Spline 컴포넌트 추가해서 사용할려고 하는데 C#스크립팅으로 되어있어서 월드 오픈시 문제는 없을까요??

**Reply by Deytooh** [reply]
@dydgnlkim Typescript 로 다시 써야돼요 ㅋㅋㅋㅋ C# 불가능해서
